prelude/cxx/compile.bzl

```diff
@@ -339,6 +372,27 @@ def _compile_single_cxx(
         output_args = output_args,
     )
 
+    if compile_pch:
+        if src_compile_cmd.cxx_compile_cmd.compiler_type == "windows":
+            cmd.add(cmd_args(compile_pch.basename_src, format="/Yc{}"))
```

Comment:
Can this just be added to `output_args` to avoid having to replicate the same conditional structure twice?

Reply:
good thinking, updated!